draft/spec/index.html

```diff
@@ -913,12 +913,12 @@ <h2>Storage Hierarchies</h2>
           hierarchies, the following restrictions apply:
         </p>
         <ol>
+          <li>There MUST be a well-defined mapping from an object identifier to a unique storage path</li>
```

Comment:
```
There MUST be a well-defined, [reproducible] mapping...
```

Reply:
Do you really want to add "reproducible"? Seems over the top to me

Reply:
yeah, given feedback so far, i would agree that adding reproducible might be a bit much.

Reply:
Perhaps "deterministic"?

Reply:
well-defined is deterministic ... see https://en.wikipedia.org/wiki/Well-defined

Reply:
agree with discussion to change from well-defined to deterministic